ctapipe/containers.py

```diff
@@ -1044,3 +1088,86 @@ class ArrayEventContainer(Container):
         "Container for calibration coefficients for the current event",
     )
     mon = Field(MonitoringContainer(), "container for event-wise monitoring data (MON)")
+
+
+class SchedulingBlockContainer(Container):
+    """Stores information about the scheduling block. This is a simplified version
+    of the SB model, only storing what is necessary for analysis."""
+
+    container_prefix = "sb"
+    id = Field(-1, "Scheduling block ID", type=np.int64)
+    type = Field(
+        SchedulingBlockType.OBSERVATION,
+        description="Type of scheduling block",
+        type=SchedulingBlockType,
+    )
+    observing_mode = Field(ObservingMode.CUSTOM, "Observing mode", type=ObservingMode)
+    pointing_mode = Field(
+        None,
+        "Pointing Mode is the coordinate frame in which the pointing position is non-moving",
+        type=CoordinateFrameType,
```

Comment:
Why is this a `CoordinateFrameType` and not a `PointingMode` enum? As mentioned, this could also be an asteroid tracking for example ... So I would really call it `PointingMode` and to keep possible extensions to things not purely related to being constant in a specific frame.

Reply:
Ah, I was just playing with the idea that they are actually the same thing and hadn't implemeted the PointingModeType, but I will likely add the PointingModeType.  The reason I was thinking about this was that this defines a CoordinateFrame that is fixed in time: e.g. DRIFT implies a fixed ALTAZ, POINTING (which I don't like because that is not the definition of pointing) implies ICRS,  and if you track the moon for example, it would imply LUNAR)

Reply:
So maybe  really we don't even need this mode, we just need a pointing_coordinate that has a frame that is implied to be fixed in time.

Reply:
The `POINTING` name comes from OGIP.

I also don't like it for the same reason of not being obvious.. We should just call them `TRACKING_ALTAZ` or `TRACKING_ICRS` or similar.